src/tracker-component.js

```diff
@@ -214,59 +95,26 @@ const getJetlorePayload = (type : string, options : Object) : Object => {
 
 let trackEventQueue = [];
 
-const track = <T>(config : Config, trackingType : TrackingType, trackingData : T) => {
+export const clearTrackQueue = (config : Config) => {
+    // $FlowFixMe
+    return trackEventQueue.filter(([ trackingType, trackingData ]) => { // eslint-disable-line array-callback-return
```

Comment:
nitpick: much cleaner to use a forEach and then `return []` imo

Reply:
Agreed. This was Zach's original implementation, and looks like it's just moved in the file. Maybe we can add a TODO instead to change this method?

Reply:
👍

Reply:
added